google-cloud-speech/src/main/java/com/google/cloud/speech/spi/v1beta1/SpeechClient.java

```diff
@@ -249,7 +278,7 @@ public final SyncRecognizeResponse syncRecognize(SyncRecognizeRequest request) {
    * try (SpeechClient speechClient = SpeechClient.create()) {
    *   RecognitionConfig config = RecognitionConfig.newBuilder().build();
    *   RecognitionAudio audio = RecognitionAudio.newBuilder().build();
-   *   Operation response = speechClient.asyncRecognize(config, audio);
+   *   AsyncRecognizeResponse response = speechClient.asyncRecognizeAsync(config, audio).get();
```

Comment:
I consider this to be a flaw of the operation name - it should not contain "async" in it. (This is the opinion of multiple other people too). I don't know what we can really do in our own layer.